test/multiverse/suites/active_support_logger/Envfile

```diff
@@ -0,0 +1,23 @@
+# This file is distributed under New Relic's license terms.
```

Comment:
I realized our `chain` instrumentation for ActiveSupport::Logger was never being tested, so I created a multiverse suite to make sure it gets checked.